future/src/ct/ct_list.cc

```diff
@@ -167,14 +167,15 @@ CtListInfo CtList::list_get_number_n_level(Gtk::TextIter iter_first_paragraph)
     int level = 0;
     while (iter_start) {
         auto ch = iter_start.get_char();
-        if (str::indexOf(_pCtMainWin->get_ct_config()->charsListbul, ch) != -1) {
+        auto& bulls_list = _pCtMainWin->get_ct_config()->charsListbul.item();
+        if (str::indexOf(bulls_list, ch) != -1) {
```

Comment:
I'd add indexOf into the class, to make class more useful, but it's up to you

Reply:
I'll add a `find` specialisation for `char` to `CtStringSplittable`

Reply:
On seconds thoughts maybe not, just realised I would have to take `char` by const ref